### PR TITLE
fix: Fix TLS hanging connection

### PR DESCRIPTION
**Pull request by zak-pawel**
Fixing TLS hanging connection as described here: https://github.com/digitalocean/go-libvirt/issues/89

resolves: https://github.com/thomasklein94/packer-plugin-libvirt/issues/11

### CODE REVIEW

**Comment by zak-pawel** [comment]
@thomasklein94 Did you have time to look at proposed fix?

**Review by thomasklein94 — APPROVED** [review]
Hi
Sorry, it was yet another busy week for me and I just missed your PR completely. This is a weird issue, and it's also weird that it's been present for years now. Nevertheless, if you've encountered this issue and you are certain this is the right fix, I'm okay with this PR.
Feel free to merge it.

**Comment by zak-pawel** [comment]
@thomasklein94 thanks, I hope I will have time in the near future to investigate it further to find out where the real problem resides.
I do not have write access to this repository, thus I cannot merge it.

**Pull request merged**
